delfin/db/api.py

```diff
@@ -362,3 +362,84 @@ def alert_source_get_all(context, marker=None, limit=None, sort_keys=None,
     """
     return IMPL.alert_source_get_all(context, marker, limit, sort_keys,
                                      sort_dirs, filters, offset)
+
+
+def port_create(context, values):
```

Comment:
Why we need Create for resources? We do not have create for volume and pool.

Reply:
We need create , There is create for pool and volume check line #111 .. I have missed create ofr ports (create interface for mutiple ports ) , added now